apps/testingaccessibility/src/hooks/use-user.ts

```diff
@@ -0,0 +1,15 @@
+import {useQuery} from 'react-query'
+
+export function useUser() {
+  const {data, status: userLoadingStatus} = useQuery(
+    '/api/auth/session',
+    async ({queryKey}) => {
+      return await fetch(queryKey[0]).then((res) => res.json())
+    },
+  )
+
+  return {
+    userLoadingStatus,
+    user: data,
+  }
+}
```

Comment:
calling this endpoint directly seems faster in my testing and gives us immediate access to the current user data that we want and can use to generate an ability set